package.json

```diff
@@ -10,10 +10,11 @@
     "scripts": {
         "clean": "rm -rf dist",
         "prebuild": "yarn run clean",
-        "build": "TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\"}' webpack --config webpack.config.ts --mode=development",
-        "build:prod": "TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\"}' NODE_ENV=production webpack --config webpack.config.ts --mode=production",
+        "build": "TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\"}' webpack --config webpack.dev.config.ts --mode=development",
+        "build:prod": "TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\"}' NODE_ENV=production webpack --config webpack.prod.config.ts --mode=production --progress",
         "build:storybook": "build-storybook",
-        "start": "node -r dotenv/config index.js",
+        "start": "webpack serve --open --config webpack.dev.config.ts --hot",
+        "start:start:prodmode": "NODE_ENV=production node -r dotenv/config index.js",
```

Comment:
`start:` part duplicated twice in prodmode . Can you please also add  `build:prod &&` step at the beginning to ensure the prod is built prior to serve. 
nit: Curios if we should print a link to use to open prod version of localhost. I think it won't open automatically